CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## [3.1.0](https://github.com/commercetools/merchant-center-application-kit/compare/v3.0.1...v3.1.0) (2018-11-28)
+
+#### ✍️ Type: Documentation
+
+- [#145](https://github.com/commercetools/merchant-center-application-kit/pull/145) chore: update docs to include build step ([@montezume](https://github.com/montezume))
```

Comment:
I wouldn't include this in the changelog

Reply:
You added the label :D I will remove.

Reply:
Removed.